src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.ts

```diff
@@ -380,7 +396,7 @@ export class CreateEditNewsComponent extends FormBaseComponent implements OnInit
     this.destroyed$.next(true);
     this.destroyed$.complete();
     if (this.formChangeSub) {
-      this.formChangeSub.unsubscribe();
+      // this.formChangeSub.unsubscribe();
```

Comment:
Do we need this unsubscribe or it must be deleted?

Reply:
uncomented this code.